README.md

```diff
@@ -1,10 +1,13 @@
-## Docker for Visual Studio Code
```

Comment:
Why remove the title? Most or all other first-party extensions have the extension name at the top of their README (examples in a comment below)

Reply:
+1 on that e.g. https://github.com/Microsoft/vscode-remote-containers Understand the desire to gain screen real estate, but it looks weird, like a book without front cover

Reply:
What stinks is the ReadMe is used in the repo and in VS Code when looking at the extension. In VS Code it is a waste of space because it duplicates the name from the VS Code header. That said, it does seem lacking when viewed in GitHub. I will add it back.